androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/PlayerControls.kt

```diff
@@ -24,11 +24,14 @@ fun PlayerControls(
     isFullScreen: Boolean,
     hasEnded: Boolean,
     speed: Float,
+    currentTime: Long,
+    totalDuration: Long,
     onPreviousClick: () -> Unit,
     onPauseToggle: () -> Unit,
     onNextClick: () -> Unit,
     onFullScreenToggle: () -> Unit,
     onSpeedSelected: (Float) -> Unit,
+    onSeekBarValueChange: (Long) -> Unit,
     customization: ControlsCustomization,
     settingsControlsCustomization: SettingsControlsCustomization,
     modifier: Modifier = Modifier
```

Comment:
nit: move title below of modifier, modifier should always be the first optional parameter